homeassistant/components/unifi/services.py

```diff
@@ -67,8 +66,8 @@ async def async_reconnect_client(hass: HomeAssistant, data: Mapping[str, Any]) -
     if mac == "":
         return
 
-    for config_entry in hass.config_entries.async_entries(UNIFI_DOMAIN):
-        if config_entry.state is not ConfigEntryState.LOADED or (
+    for config_entry in hass.config_entries.async_loaded_entries(UNIFI_DOMAIN):
+        if (
             ((hub := config_entry.runtime_data) and not hub.available)
```

Comment:
This check is a bit dodgy TBH, if we're not sure the hub is there, we should handle that, and if we are sure, this should be rewritten as:
(not (hub := config_entry.runtime_data).available)

Reply:
Integration will abort setup if no connection can be made so hub can not be None
https://github.com/home-assistant/core/blob/6d6dfce7d13195eed0ae79a485ea392f444aae71/homeassistant/components/unifi/__init__.py#L51